src/org/thoughtcrime/securesms/connect/DcEventCenter.java

```diff
@@ -62,33 +74,54 @@ public void removeObservers(DcEventDelegate observer) {
         }
     }
 
+    public void removeMultiAccountObserver(DcEventDelegate observer) {
+        synchronized (MULTI_LOCK) {
+            for(Integer eventId : multiObservers.keySet()) {
+                ArrayList<DcEventDelegate> idObservers = multiObservers.get(eventId);
+                if (idObservers != null) {
+                    idObservers.remove(observer);
+                }
+            }
+        }
+    }
+
+    public void sendToMultiAccountObservers(@NonNull DcEvent event) {
```

Comment:
```suggestion
    private void sendToMultiAccountObservers(@NonNull DcEvent event) {
```

Reply:
sendToObservers() is also public, but in fact not used anywhere outside of DcEventCenter, maybe it is used for some tests :thinking: